src/features/core/cookieConsent/OneTrust/browser.js

```diff
@@ -6,9 +6,10 @@ class OneTrust {
     // If user does not load onetrust sdk before loading rudderstack sdk
     // we will not be filtering any of the destinations.
     if (!window.OneTrust || !window.OnetrustActiveGroups) {
-      throw new Error(
+      logger.error(
         'OneTrust resources are not accessible. Thus all the destinations will be loaded',
       );
+      return;
```

Comment:
Returning from here alone may not be sufficient for the upward logic to work smoothly. Have you tested this scenario?

Reply:
I did but I will check again

Reply:
Addressed